### PR TITLE
PUD-657: Update the flush script

### PR DESCRIPTION
**Pull request by dazoakley**
* Don't error out if the port-forward is already present
* Check for system dependencies
* Also clear down the S3 bucket

### CODE REVIEW

File: scripts/flush-dev-db.sh [hunk 1]
```diff
@@ -1,18 +1,49 @@
...
+# Check dependencies
+check_dep "psql" "brew install postgresql"
+check_dep "jq" "brew install jq"
+check_dep "aws" "brew install awscli"
+check_dep "kubectl" "asdf install kubectl 1.19.15"
+
+# check for the ENV variable
+set +u
+if [[ -z "${ENV}" ]]; then
+  echo "You must set the ENV variable - allowed values: [dev, preprod, prod]"
```
Comment (philsegal) [inline scripts/flush-dev-db.sh:20]: prod seems pretty gnarly/dangerous. surely we cant/shouldnt have access to prod?

Comment (dazoakley) [reply scripts/flush-dev-db.sh:20]: Fair point - will restrict to dev or pre-prod 👍

File: scripts/flush-dev-db.sh [hunk 2]
```diff
@@ -1,18 +1,51 @@
...
+
+# Check dependencies
+check_dep "psql" "brew install postgresql"
+check_dep "jq" "brew install jq"
+check_dep "aws" "brew install awscli"
+check_dep "kubectl" "asdf install kubectl 1.19.15"
+
+# check for the ENV variable
+set +u
+if [[ ! "${ENV}" =~ ^(dev|preprod)$ ]]; then
```
Comment (AndyNewton) [inline scripts/flush-dev-db.sh:19]: Personally I'd prefer this and `port-forward-db` to both use a CL arg for env not an environment variable - but also they should be consistent with one another.

File: scripts/flush-dev-db.sh [hunk 3]
```diff
@@ -1,18 +1,51 @@
 #!/bin/bash
```
Comment (AndyNewton) [inline scripts/flush-dev-db.sh:1]: For me the name of this script is a little misleading: I could do without 'dev' in the name: suggest to me that it acts on the dev DB only.

**Review by AndyNewton — COMMENTED** [review]
Couple of comments for discussion.

**Review by philsegal — APPROVED** [review]

**Pull request merged**